docs/extend/modular-resources/examples/triton.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Add a Triton MLModel Modular Service"
+linkTitle: "Triton MLModel Modular Service"
+weight: 70
+type: "docs"
+description: "Add an ML model modular service backed by NVIDIA's Triton Server."
+tags: ["ml", "model training", "services"]
+# SMEs: Abe Winter, Andrew Morrow
+---
+
+Viam provides a MLModel service resource backed by NVIDIA's [Triton Inference Server](https://developer.nvidia.com/triton-inference-server), open-source software that makes AI model deployment and execution seamless and standardized.
+Configure this MLModel service as a [modular resource](/extend/modular-resources/) on your robot with a [`jetson` board](/components/board/jetson/) to deploy MLModels to your robot faster while consuming less compute power.
+
+Usage information is also available on [GitHub](https://github.com/viamrobotics/viam-mlmodelservice-triton).
+
+## Requirements
+
+A NVIDIA Jetson Orin board with the following installed:
+
+1. [Jetpack 5](https://developer.nvidia.com/embedded/jetpack-sdk-502)
```

Comment:
The link here is to a specific version, but really it just needs to be Jetpack 5, so https://developer.nvidia.com/embedded/jetpack is probably better. Eventually we are going to require JetPack 6.

Reply:
Sounds good to me, thanks! Changed